pre_commit/commands/run.py

```diff
@@ -77,7 +77,7 @@ def _run_single_hook(filenames, hook, repo, args, skips, cols):
             'replacement.'.format(hook['id'], repo.repo_config['repo']),
         )
 
-    if hook['id'] in skips:
+    if hook['id'] in skips or (hook['alias'] and hook['alias'] in skips):
```

Comment:
if skips can never contain the empty string I think you can drop the first part of this condition?

if skips can contain the empty string, let's fix skips

Reply:
The condition here is that, alias, by default, is an empty string...

Reply:
hmm, ok, but skips can't be empty strings. OK. I get it.

Reply:
fixed.